packages/cli/src/aws/update-app.test.ts

```diff
@@ -176,6 +176,68 @@ describe('update-app command', () => {
       })
     );
 
+    // Mock the readdirSync to list files to replace variables
+    (fs.readdirSync as jest.Mock).mockImplementation((folderName) => {
+      if (folderName.endsWith('dist')) {
+        return [{ name: 'js', isDirectory: () => true, isFile: () => false }];
+      }
+      return [
+        { name: 'main.js', isDirectory: () => false, isFile: () => true },
+        { name: 'nonejsfile', isDirectory: () => false, isFile: () => true },
```

Comment:
adds coverage for https://sonarcloud.io/code?id=medplum_medplum&pullRequest=1944&selected=medplum_medplum%3Apackages%2Fcli%2Fsrc%2Faws%2Fupdate-app.ts&line=97